tests/test_series.py

```diff
@@ -1520,3 +1522,5 @@ def test_updated_astype() -> None:
         pd.Series,
         Timestamp,
     )
+
+    # check(assert_type(pd.Series([Decimal(x) for x in [1,2,3]]).astype(DecimalDtype()), pd.Series), Decimal)
```

Comment:
This should work.
```python
    orseries = pd.Series([Decimal(x) for x in [1, 2, 3]])
    newtype: ExtensionDtype = DecimalDtype()
    decseries = orseries.astype(newtype)
    check(
        assert_type(decseries, pd.Series),
        pd.Series,
        Decimal,
    )
```
I tested this locally.  I think we have to do it this way because where we import `DecimalDtype` from doesn't really pass the type checking, so it gets confused, but saying that `DecimalDtype` is `ExtensionDtype` is OK.

Reply:
thanks Sir Done with all the changes